### PR TITLE
fixed : logo clicks redirect to homepage

### PR DESCRIPTION
**Pull request by chandra1899**
issue:- #215

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ glob.sync('./models/**/*.js').forEach(function (file) {
 
 // Start our app!
 const app = require('./app');
-app.set('port', process.env.PORT || 8888);
+app.set('port', 8888);
```
Comment (quan-nh2) [inline server.js:32]: Need to load `port` from env first

Comment (chandra1899) [reply server.js]: > Need to load `port` from env first

Updated .....

**Review by salahlalami — CHANGES_REQUESTED** [review]
@chandra1899 Thank you , please add this svg logo instead of png. , should be same siez

idurar-crm-erp

**Comment by chandra1899** [comment]
> @chandra1899 Thank you , please add this svg logo instead of png. , should be same siez
> 
> idurar-crm-erp

added 👍

**Comment by polymahh** [comment]
@chandra1899 is this logo working correctly when the navigation is closed ?
@salahlalami navigation needs two files one for icon and one for text

**Review by salahlalami — CHANGES_REQUESTED** [review]
Thank you @polymahh  , 
@chandra1899 please use this two svg (text and icon) 
logo-text
logo-icon

**Comment by chandra1899** [comment]
> Thank you @polymahh , @chandra1899 please use this two svg (text and icon) logo-text logo-icon

pull request updated

**Review by salahlalami — APPROVED** [review]
Thank you @chandra1899

**Pull request merged**